packages/realm/bindgen/src/templates/jsi.ts

```diff
@@ -333,6 +336,9 @@ function convertPrimFromJsi(addon: JsiAddon, type: string, expr: string): string
     case "uint64_t":
       return `bigIntToU64(_env, jsi::Value(_env, ${expr}))`;
 
+    case "std::chrono::milliseconds":
+      return `std::chrono::milliseconds(bigIntToU64(_env, jsi::Value(_env, ${expr})))`;
```

Comment:
Thinking more about this change. We should have been able to ignore this primitive all together, by simply not including it in the `js_opt_in_spec`. Did bindgen fail if you didn't put this code in?